app/code/community/Bolt/Boltpay/Block/Checkout/Boltpay.php

```diff
@@ -677,10 +677,12 @@ public function getPublishableKeyForRoute()
         $checkoutType = static::CHECKOUT_TYPE_MULTI_PAGE;
         if ($routeName === 'adminhtml') {
             $checkoutType = static::CHECKOUT_TYPE_ADMIN;
-        } else if ( ($routeName === 'firecheckout') || ($routeName === 'checkout' && $controllerName === 'onepage') ) {
+        } else if ($routeName === 'checkout' && $controllerName === 'onepage') {
```

Comment:
It does not remove any official support. I moved a 3rd party support to the observer model where is it place. We can easily add new 3rd party checkouts due the event. It will not break any production code because we have an event now.